raiden/tests/unit/test_pfs_integration.py

```diff
@@ -697,6 +697,7 @@ def test_routing_in_direct_channel(happy_path_fixture, our_signer, one_to_n_addr
         "raiden.routing.query_address_metadata"
     ) as pfs_user_request:
         pfs_route_request.return_value = None, [], "feedback_token"
+        pfs_user_request.return_value = None
```

Comment:
Shouldn't `get_best_routes_pfs` return a tuple?

Reply:
That's the return value for the second patched function (`raiden.routing.query_address_metadata`)

Reply:
Ah, indeed...